'make' command is failing on AttributeError: 'dict' object has no attribute 'has_key'

```
➜  linz-lds-bde-schema/code (master) ✗ make
sed -e 's/@@VERSION@@/1.7.0dev/;s|@@REVISION@@|1.5.0-22-g3b67091|' sql/04-lds_layer_functions.sql.in > sql/04-lds_layer_functions.sql
sed -e 's/@@VERSION@@/1.7.0dev/;s|@@REVISION@@|1.5.0-22-g3b67091|' sql/06-bde_ext_functions.sql.in > sql/06-bde_ext_functions.sql
sed -e 's/@@VERSION@@/1.7.0dev/;s|@@REVISION@@|1.5.0-22-g3b67091|' sql/07-lds_version.sql.in > sql/07-lds_version.sql
rm -f sql/95-lds_comments.sql
python doc/tools/comment-extraction.py doc/lds-full-landonline-data-dictionary-and-models.md > sql/95-lds_comments.sql
Traceback (most recent call last):
  File "doc/tools/comment-extraction.py", line 240, in <module>
    if mappings.has_key(name):
AttributeError: 'dict' object has no attribute 'has_key'
Makefile:84: recipe for target 'sql/95-lds_comments.sql' failed
make: *** [sql/95-lds_comments.sql] Error 1
```

```
➜  linz-lds-bde-schema/code (master) ✗ python --version
Python 3.6.8

```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -6,6 +6,7 @@ this file.
 ## 1.7.0dev - YYYY-MM-DD
 ### Added
 - linz-lds-bde-schema-publish script (#176)
+- Build support for python3 (#181)
 
 ## 1.6.1 - 2019-08-14
 ### Fixed

--- a/doc/tools/comment-extraction.py
+++ b/doc/tools/comment-extraction.py
@@ -237,7 +237,7 @@ with open(sys.argv[1]) as fp:
           else:
             break
         name = name[:len(name)-1].replace('\n', '')
-        if mappings.has_key(name):
+        if name in mappings:
           table_name = mappings[name]
         else:
           table_name = name.lower()

